src/Receipts/ReceiptResponse.php

```diff
@@ -98,26 +118,59 @@ public function getIsRetryable(): ?bool
     }
 
     /**
-     * @return string
+     * @return string|null
      */
     public function getLatestReceipt(): ?string
     {
         return $this->latestReceipt;
     }
 
     /**
-     * @return array|ReceiptInfo[]
+     * @return array|LatestReceiptInfo[]
      */
-    public function getLatestReceiptInfo(): array
+    public function getLatestReceiptInfo(): ?array
```

Comment:
there seems to be a type mismatch with the docblock here

Reply:
Fixed